conf/app.ini

```diff
@@ -113,6 +113,8 @@ LOCAL_ROOT_URL = %(PROTOCOL)s://%(HTTP_ADDR)s:%(HTTP_PORT)s/
 DISABLE_SSH = false
 ; Whether use builtin SSH server or not.
 START_SSH_SERVER = false
+; Username to use for builtin SSH server. If blank, then it is the value of RUN_USER.
+BUILTIN_SSH_USER =
```

Comment:
IMO is better to use name like `BUILTIN_SSH_SERVER_USER`.

Reply:
Should I change it?

Reply:
I have changed it.